ccpp/CMakeLists.txt

```diff
@@ -178,48 +178,6 @@ endif (DYN32)
 ADD_DEFINITIONS(-DMOIST_CAPPA -DUSE_COND -DNEMS_GSM)
 ADD_DEFINITIONS(-DINTERNAL_FILE_NML)
 
-#------------------------------------------------------------------------------
```

Comment:
This *should* be ok because `ccpp/{framework,physics,data,driver}` each have their own logic. Have you checked that the computational performance is not affected by this change?